Source/Extensions/Result+Value.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Result+Value.swift
+//  Swish
+//
+//  Created by Samuel Goodwin on 3/26/19.
+//  Copyright © 2019 thoughtbot. All rights reserved.
+//
+
+import Foundation
+
+extension Result {
+  public var value: Success? {
```

Comment:
I don’t think Swish should be extending the standard library’s `Result` here. Given that this is already a breaking change, I think it would be fine to remove this extension.